benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -87,23 +87,23 @@ msgstr ""
 
 #: benefits/core/migrations/0002_sample_data.py:135
 msgid "eligibility.pages.index.mst.label"
```

Comment:
I wonder if this `mst` should be `cc` or `mst_cc` or something to clarify that this means Courtesy Card flow